data.js

```diff
@@ -1044,6 +1044,14 @@ module.exports = [
     url: "https://github.com/Temoto-kun/avtomat",
     source: "https://raw.githubusercontent.com/Temoto-kun/avtomat/master/src/avtomat.js"
   },
+  {
+    name: "svelte-coordinate-conversion",
+    github: "rozek/svelte-coordinate-conversion",
+    tags: ["coordinate-conversions","local-coordinates","viewport-coordinates","document-coordinates"],
+    description: "converts coordinates between viewport, document and element coordinate systems (not only in Svelte)",
+    url: "https://github.com/rozek/svelte-coordinate-conversion",
+    source: "https://raw.githubusercontent.com/rozek/svelte-coordinate-conversion/main/src/svelte-coordinate-conversion.ts"
```

Comment:
From the readme - emphasis on "ready-to-use", which means you should not link to a TypeScript source file:

> The 'source' attribute must point to a single-file, or an array of files that can be concatenated to form your complete library in a ready-to-use form. The 'source' must not be minified (see below)

Reply:
> From the readme - emphasis on "ready-to-use", which means you should not link to a TypeScript source file:
> 
> 
> 
> > The 'source' attribute must point to a single-file, or an array of files that can be concatenated to form your complete library in a ready-to-use form. The 'source' must not be minified (see below)